NhanNT-Kozocom/react-graphql-demo
# feature: create add author page

#### Describe your changes
- Create add author page
#### Issue ticket number and link
- N/A
#### Range of influence to consider
- Create author page
#### How to test and results
| Page | Result |
|----------|----------|
| Edit author | 
![image](https://github.com/NhanNT-Kozocom/react-graphql-demo/assets/145307468/0a9b71f6-13ad-49b9-acd1-e76543810a72) |
| Confirm delete 1 | 
![image](https://github.com/NhanNT-Kozocom/react-graphql-demo/assets/145307468/ce418123-2029-44aa-976d-43d8748408ec) |
| Delete success | ![image](https://github.com/NhanNT-Kozocom/react-graphql-demo/assets/145307468/b103548a-8ea7-44d5-a369-b551867bae47) |
#### Note
- N/A
